adserver/tasks.py

```diff
@@ -248,3 +248,23 @@ def daily_update_uplift(day=None):
             UpliftImpression.objects.filter(pk=impression.pk).update(
                 **{impression_type: values["total"]}
             )
+
+
+@app.task()
+def daily_update_all_reports(day=None):
```

Comment:
A bit confusing to pass in a day and index the prior. Probably need to change this name a bit:

```suggestion
def update_previous_day_reports(day=None):
```

Reply:
I'll take this change although I have to change more than this one line.